packages/app/src/debug/DebugNewRelevantRunBar.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div
+    v-if="data.status && data.runNumber"
+    data-cy="newer-relevant-run"
+    class="w-full px-24px pb-16px gap-y-1"
+  >
+    <ul
+      id="metadata"
+      class="rounded flex flex-row bg-indigo-100 border-indigo-100 p-12px gap-x-2 items-center whitespace-nowrap children:flex children:items-center"
+    >
+      <li>
+        <DebugRunNumber
+          :status="data.status"
+          :value="data.runNumber"
+        />
+      </li>
+      <li class="font-medium text-sm text-gray-800 truncate">
+        {{ data.commitInfo?.summary }}
+      </li>
+      <li class="font-normal text-sm truncate">
+        <DebugPendingRunCounts
+          v-if="data.status === 'RUNNING'"
+          :total-skipped="data.totalSkipped || 0"
+          :total-failed="data.totalFailed || 0"
+          :total-passed="data.totalPassed || 0"
+          :total-tests="data.totalTests || 0"
+        />
+        <span
```

Comment:
I assume this isn't using `ExternalLink` because there are some other things that need to happen other than simply opening the page for the run.

Reply:
This is not intended to open the browser but rather update the currently-viewed run within the DebugContainer, so it's not really a link.

For reference, the requirement:

> The text at the end of the banner that originally indicated the number of specs completed in the in-progress run will now be a button styled as a link that says "View run". Clicking the link will update the page to now show that run's information in the Debug page.

As the requirement states, this really should be a link-styled button so I'll get that change made